pom.xml

```diff
@@ -1162,15 +1163,6 @@
                 </plugins>
             </build>
         </profile>
-        <profile>
-            <id>jdk9-</id>
```

Comment:
I think this shouldn't be removed but renamed to jdk10- and changed accordingly.

AFAIK, we can't run the tests with JDK 11 yet.

Reply:
Hi @gsmet, does the project support JDK 11? It seems I can build it on JDK 11.

I can specifically disable the OSGI module if you want?

Reply:
Yes, we should be able to build HV with JDK 11.

If what you did also works with JDK 11 then that's perfect. I thought Karaf did not support JDK 11 yet.

Reply:
Hi @gsmet PR updated, thanks!

Reply:
@gsmet How is the removal of JavaFX handled when building on 11? Just learned that first releases of FX are available as snapshots: http://mail.openjdk.java.net/pipermail/openjfx-dev/2018-July/022088.html. But it seems you already found a way around it?

Reply:
@gunnarmorling for now, I download the artifacts from the website. I'm pondering migrating to using the published artifacts, just not very excited to add a snapshot repository. But if it's only for JDK 11, I suppose it's OK.